plugins/infinispan100/src/main/java/org/radargun/service/Infinispan100ServerService.java

```diff
@@ -0,0 +1,128 @@
+package org.radargun.service;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.Clustered;
+import org.radargun.traits.ProvidesTrait;
+import org.radargun.utils.Utils;
+
+@Service(doc = InfinispanServerService.SERVICE_DESCRIPTION)
+public class Infinispan100ServerService extends Infinispan80ServerService {
+
+
+   @Property(doc = "Use Serverng instead of Wildfly. Default is false.")
+   private boolean serverng = false;
+
+   @Property(doc = "Rest server which will be used to retrieve cache manager information")
```

Comment:
Ok, so I will maintain only `restServer` because we need this parameter, restServer parameter will indicate which server we will use to request cluster informations

Reply:
I guess I'd prefer to have a boolean property if this URL can be determined. The issue I see is that you have to know the URL of one of the servers in a benchmark, which has been a chicken-and-egg problem in some situations.